src/ebay-skeleton/ebay-skeleton-avatar.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { ComponentProps, FC } from 'react'
+import classNames from 'classnames'
+import { NativeTags } from './nativeTags'
```

Comment:
We should use the react type instead:

```
    as?: keyof React.JSX.IntrinsicElements;
```

But I think skin supports `div` and `span` only